app/data/models/application.js

```diff
@@ -1,4 +1,11 @@
-const createAgreementNumber = require('../../lib/create-agreement-number')
+const { createApplicationReference } = require('../../lib/create-reference')
+
+const afterCreate = async (applicationRecord, _) => {
```

Comment:
Given that we are now exporting it. Wonder if we'd be better to name this function by what it does, "update the application record", rather than when it is called by sequelize, "after it's created a record".

Reply:
Yeah good call - will update the name